spec/support/github_service_helper.rb

```diff
@@ -0,0 +1,217 @@
+require 'octokit'
+require 'github_service'
+require 'github_service/issue'
+
+module GithubServiceHelper
```

Comment:
Out of curiosity, why use mocks when you can use VCR?

Reply:
I don't like VCR?

But in all seriousness, I do find VCR to be a heavy handed approach and try to avoid it when possible, since it creates a lot of bloat and churn in repos.  And it usually requires someone having specific access to an particular account to re-record when things change.

Regarding bloat and for some context, we would have a blob of this size in a yaml size somewhere just for fetching a single issue:

https://developer.github.com/v3/issues/#get-a-single-issue

Doing it this way, when things change (say, we update `octokit`), we just need to change a few lines if the expected JSON response has changed.  This is no better or worse (in my opinion) with what you would have to do with VCR.

* * *

To be fair, I don't think VCR is completely with out merit, I just try to avoid using it when possible.  I also find the `Faraday` mocking framework pretty nice and robust for my needs.